website/assets/css/common.css

```diff
@@ -97,6 +97,14 @@ header .news:hover:not(.active) {
   color: red;
   border-bottom: 2pt solid gray;
 }
+header .workshops {
```

Comment:
The "News" tab is red and the "Workshops" tab is purple. I think I prefer having all links be black or at most one link being a different color? What do you think?

Reply:
I don't have strong opinions. Changed to black.